### PR TITLE
Runtime: correct some whitespace issues (NFC)

### PR DESCRIPTION
**Pull request by compnerd**
Replace some hard tabs with spaces and remove some bleeding whitespace.

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @mikeash @grynspan

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Review by mikeash — APPROVED** [review]

**Review by grynspan — APPROVED** [review]

File: include/swift/Runtime/HeapObject.h [hunk 1]
```diff
@@ -159,7 +159,7 @@ static inline T *swift_cxx_newObject(Args &&... args) {
 /// Destruct and deallocate an instance of type \c T.
 ///
 /// \param ptr A pointer to an instance of type \c T previously created with a
-/// 	call to \c swift_cxx_newObject().
+///     call to \c swift_cxx_newObject().
```
[the remark this answers is not in the document]
Comment (compnerd) [reply include/swift/Runtime/HeapObject.h]: Fixed!  I wasn't sure about it; I just expanded it out as if the tab alignment was desired.

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Pull request merged**